### PR TITLE
added swim

### PR DESCRIPTION
**Pull request by karl-police**
has to be tested not sure

### CODE REVIEW

**Comment by kurozael** [comment]
Please don’t submit pull requests for code that is not tested, all pull requests must be tested thoroughly :)

File: upload/garrysmod/gamemodes/clockwork/framework/sh_kernel.lua [hunk 1]
```diff
@@ -727,6 +727,10 @@ function Clockwork:HandlePlayerSwimming(player)
 		end;
 		
 		player.m_bInSwim = true;
+		
+		if (player.m_bInSwim) then
```
Comment (nixerus) [inline upload/garrysmod/gamemodes/clockwork/framework/sh_kernel.lua]: Surely as this variable is defined two lines above, it'd be redundant to do a check here for if the value is true?

File: upload/garrysmod/gamemodes/clockwork/framework/sh_kernel.lua [hunk 2]
```diff
@@ -5474,4 +5479,4 @@ function Clockwork.kernel:ParseData(text)
 	end;
 	
 	return Clockwork.config:Parse(text);
-end;
\ No newline at end of file
+end;
```
Comment (nixerus) [inline upload/garrysmod/gamemodes/clockwork/framework/sh_kernel.lua]: Why is there an additional end here?

Comment (nixerus) [reply upload/garrysmod/gamemodes/clockwork/framework/sh_kernel.lua]: Ah nevermind just realized that it was Git being annoying again

**Comment by moschetto** [comment]
I'm closing this for now due to no prior testing being done. If someone implements this and finds it sufficient, you can comment below and someone'll re-check it.

**Pull request closed without merging**